go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	k8s.io/api v0.20.4
 	k8s.io/apimachinery v0.20.4
 	k8s.io/client-go v0.20.4
-	k8s.io/klog v1.0.0
+	k8s.io/klog/v2 v2.4.0
```

Comment:
Can it be 2.9 as NFD https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/go.mod#L20